suite/tests/client-interface/drreg-test.c

```diff
@@ -163,10 +159,7 @@ GLOBAL_LABEL(FUNCNAME:)
         jmp      epilog
      epilog:
         add      REG_XSP, FRAME_PADDING /* make a legal SEH64 epilog */
-        pop      REG_XDI
-        pop      REG_XSI
-        pop      REG_XBP
-        pop      REG_XBX
+        POP_CALLEE_SAVED_REGS()
         ret
 #elif defined(ARM)
         b        test1
```

Comment:
Seems like the arm side should also push and pop the callee-saved.

Reply:
I'd have to add this blindly, can't test it (except on GitHub)